static/app/views/discover/tags.tsx

```diff
@@ -152,9 +180,29 @@ class Tags extends Component<Props, State> {
 
     if (tags.length > 0) {
       return (
-        <TagFacetsList>
-          {tags.map((tag, index) => this.renderTag(tag, index))}
-        </TagFacetsList>
+        <Fragment>
+          <StyledTagFacetList>
+            {tags.map((tag, index) => this.renderTag(tag, index))}
+          </StyledTagFacetList>
+          {hasMore &&
+            (loading ? (
+              this.renderPlaceholders()
+            ) : (
```

Comment:
Does `Show More` show if there is an error? 🤔

Reply:
No, if there's an error then there's a conditional above that renders an error state for the whole sidebar